<launch>
  <group ns="camera_center">
    <node pkg="uvc_camera" name="camera0" type="uvc_camera_node" output="screen">
      <param name="device" type="string" value="/dev/video0"/>
      <param name="width" type="int" value="640"/>
      <param name="height" type="int" value="480"/>
      <param name="pixel_format" value="mjpeg"/>
      <param name="camera_frame_id" value="usb_cam"/>
      <param name="io_method" value="mmap"/>
      <param name="frame_id" value="camera_center"/>
      <param name="camera_info_url" type="string" value="file:///home/{user_name}/.../camera_center.yaml"/>
    </node>

    <node name="image_view_center" pkg="image_view" type="image_view" output="screen">
      <remap from="image" to="image_raw">

      </remap>
    </node>
  </group>

  <group ns="camera_left">
    <node pkg="uvc_camera" name="camera1" type="uvc_camera_node" output="screen">
      <param name="device" type="string" value="/dev/video6"/>
      <param name="width" type="int" value="640"/>
      <param name="height" type="int" value="480"/>
      <param name="pixel_format" value="mjpeg"/>
      <param name="camera_frame_id" value="usb_cam"/>
      <param name="io_method" value="mmap"/>
      <param name="frame_id" value="camera_left"/>
      <param name="camera_info_url" type="string" value="file:///home/{user_name}/.../camera_left.yaml"/>
    </node>

    <node name="image_view_left" pkg="image_view" type="image_view" output="screen">
      <remap from="image" to="image_raw">
      </remap>
    </node>
  </group>

  <group ns="camera_right">
    <node pkg="uvc_camera" name="camera2" type="uvc_camera_node" output="screen">
      <param name="device" type="string" value="/dev/video8"/>
      <param name="width" type="int" value="640"/>
      <param name="height" type="int" value="480"/>
      <param name="pixel_format" value="mjpeg"/>
      <param name="camera_frame_id" value="usb_cam"/>
      <param name="io_method" value="mmap"/>
      <param name="frame_id" value="camera_right"/>
      <param name="camera_info_url" type="string" value="file:///home/{user_name}/.../camera_right.yaml"/>
    </node>

    <node name="image_view_right" pkg="image_view" type="image_view" output="screen">
      <remap from="image" to="image_raw">
      </remap>
    </node>
  </group>

</launch>

